sirepo/package_data/static/js/controls.js

```diff
@@ -103,13 +109,63 @@ SIREPO.app.controller('ControlsController', function(appState, controlsService,
         };
     }
 
+    function updateFromMonitorValues(monitorValues) {
+        monitorValues.forEach((value) => {
+            // TODO(e-carlin): How can I eliminate the timeout. Needed because
+            // on first load self.watches needs to be populated before there is
+            // anyone listening to the broadcast
+            $timeout(function () {
```

Comment:
@moellep any idea how to eliminate this timeout? The sr-point-data-* listeners needs to be registered so the broadcast has somewhere to go. That is done by populating self.watches and rendering the bpm plots. But, the handleStatus comes in before models are loaded which leads to a race condition.

Reply:
This is tricky because the UI is building itself based on the runStatus results, so if it broadcasts right away, the directives won't be listening. For now I added a call to panelState.waitForUI() which is really just a $timeout call eventually.